examples/howto/updating_priors.myst.md

```diff
@@ -10,56 +10,61 @@ kernelspec:
   name: python3
 ---
 
-# Updating priors
+(updating-priors)=
+# Updating Priors
+
+:::{post} January, 2017
+:tags: priors
+:category: intermediate, how-to
```

Comment:
Can we find who the original author was?

Reply:
No, I could not track the original author via git blame.